### ISSUE TITLE
AssertionError: isempty(XML_GLOBAL_ERROR_STACK)

### ISSUE THREAD

**Issue by duyanran** [felipenoris/XLSX.jl#143]
hi could anyone tell me why is this happen：
```
using XLSX
m = XLSX.readxlsx("test.xlsx")
```
```
ERROR: AssertionError: isempty(XML_GLOBAL_ERROR_STACK)
Stacktrace:
 [1] macro expansion at C:\Users\duyanran\.julia\packages\EzXML\ZNwhK\src\error.jl:49 [inlined]
 [2] readxml(::ZipFile.ReadableFile) at C:\Users\duyanran\.julia\packages\EzXML\ZNwhK\src\document.jl:154
 [3] internal_xml_file_read(::XLSX.XLSXFile, ::String) at C:\Users\duyanran\.julia\packages\XLSX\tfTrI\src\read.jl:427
 [4] open_or_read_xlsx(::String, ::Bool, ::Bool, ::Bool) at C:\Users\duyanran\.julia\packages\XLSX\tfTrI\src\read.jl:207
 [5] readxlsx(::String) at C:\Users\duyanran\.julia\packages\XLSX\tfTrI\src\read.jl:37
 [6] top-level scope at REPL[26]:1
```
 all Testing XLSX tests passed.

really need some help and i saw other friends have the same trouble.

**Comment by felipenoris** [comment]
@duyanran , looks like an internal EzXML error. It is most likely that the Excel file is malformed. Can you share the file you`re trying to read? Does this happen with any Excel file you try to read?

**Comment by guilhermebodin** [comment]
Same thing here 
```julia
using XLSX
XLSX.readxlsx("/Users/guilhermebodin/Downloads/IPDO-01-01-2018.xlsm")


```

**Comment by felipenoris** [comment]
I could find the cause of the error. But it's not an easy fix. https://julialang.slack.com/archives/C674VR0HH/p1600363989121700?thread_ts=1600311174.098200&cid=C674VR0HH

**Comment by MillironX** [comment]
@felipenoris Can you summarize the contents of the slack link? I'm not authorized to view that post, but am having this issue, too.

**Comment by felipenoris** [comment]
@MillironX in short, as far as I could understand the problem, some Excel plugins use custom XML namespaces that EzXML can't parse.
